assets/models/system/stabilityai-stable-diffusion-xl-base-1.0/description.md

```diff
@@ -0,0 +1,88 @@
+Real time|<a href="https://aka.ms/azureml-infer-sdk-text-to-image" target="_blank">text-to-image-online-endpoint.ipynb</a>|<a href="https://aka.ms/azureml-infer-cli-text-to-image" target="_blank">text-to-image-online-endpoint.sh</a>
+Batch |<a href="https://aka.ms/azureml-infer-batch-sdk-text-to-image" target="_blank">text-to-image-batch-endpoint.ipynb</a>|<a href="https://aka.ms/azureml-infer-batch-cli-text-to-image" target="_blank">text-to-image-batch-endpoint.sh</a>
+
+<h3> Inference with <a href="https://learn.microsoft.com/en-us/azure/ai-services/content-safety/studio-quickstart", target="_blank">Azure AI Content Safety (AACS)</a> samples </h3>
+
+Inference type|Python sample (Notebook)
+|--|--|
+Real time|<a href="https://aka.ms/azureml-infer-sdk-safe-text-to-image" target="_blank">safe-text-to-image-online-deployment.ipynb</a>
+Batch |<a href="https://aka.ms/azureml-infer-batch-sdk-safe-text-to-image" target="_blank">safe-text-to-image-batch-endpoint.ipynb</a>
+
+# Sample input and output
+
+### Supported Parameters
+
+- num_inference_steps: The number of de-noising steps. More de-noising steps usually lead to a higher quality image at the expense of slower inference, defaults to 50.
+- guidance_scale: A higher guidance scale value encourages the model to generate images closely linked to the text `prompt` at the expense of lower image quality. Guidance scale is enabled when `guidance_scale > 1`, defaults to 7.5.
+
+> These `parameters` are optional inputs. If you need support for new parameters, please file a support ticket.
+
+### Sample input
+
+```json
+{
+   "input_data": {
+        "columns": ["prompt"],
+        "data": ["a photograph of an astronaut riding a horse"],
+        "index": [0],
+        "parameters": {
+            "num_inference_steps": 50,
+            "guidance_scale": 7.5
+        }
+    }
+}
+```
+
+### Sample output
+
+```json
+[
+    {
+        "prompt": "a photograph of an astronaut riding a horse",
+        "generated_image": "image",
+        "nsfw_content_detected": null
+    }
+]
+```
+
+> Note:
+>
+> - "image" string is in base64 format.
+> - The `stabilityai-stable-diffusion-xl-base-1.0` model doesn't check for the NSFW content in generated image. We highly recommend to use the model with <a href="https://learn.microsoft.com/en-us/azure/ai-services/content-safety/studio-quickstart" target="_blank">Azure AI Content Safety (AACS)</a>. Please refer sample <a href="https://aka.ms/azureml-infer-sdk-safe-text-to-image" target="_blank">online</a>  and <a href="https://aka.ms/azureml-infer-batch-sdk-safe-text-to-image" target="_blank">batch</a> notebooks for AACS integrated deployments.
+
+#### Visualization for the prompt - "a photograph of an astronaut riding a horse"
+
+<img src="https://automlcesdkdataresources.blob.core.windows.net/finetuning-image-models/images/Model_Result_Visualizations(Do_not_delete)/output_stabilityai_stable_diffusion_2_1.png" alt="stabilityai_stable_diffusion_2_1 visualization">
```

Comment:
Please use the right viz image. Not the viz image from stable diffusion 2.1

Reply:
please generate the sample image using the same input from stable diffusion 2.1 and use that here

Reply:
added!